### PR TITLE
Sidebar creatitem rename & upgrades

### LINKED ISSUE #2139: Sidebar: "CreateItem" changes
More generic naming, configurable icon

### PR DESCRIPTION
**Pull request by oddvernes**
resolves #2139

### CODE REVIEW

File: packages/eds-lab-react/src/components/SideBar/ActionButton/index.tsx [hunk 1]
```diff
@@ -89,39 +89,41 @@ const Tooltip = styled(EDSTooltip)`
   }
 `
 
-export type CreateItemProps = {
-  createLabel: string
-  onCreate: () => void
+export type ActionButtonProps = {
+  actionLabel: string
+  actionIcon: IconData
```
Comment (mimarz) [inline packages/eds-lab-react/src/components/SideBar/ActionButton/index.tsx:94]: Suggest dropping the the pre "action" in wording of prop name.
```suggestion
  label: string
  icon: IconData
```

**Review by mimarz — CHANGES_REQUESTED** [review]
Suggested wording.

File: packages/eds-lab-react/src/components/SideBar/ActionButton/index.tsx (hunk 1 above)
Comment (oddvernes) [reply packages/eds-lab-react/src/components/SideBar/ActionButton/index.tsx]: Done

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**